hazelcast/proxy/map.py

```diff
@@ -898,12 +898,9 @@ class MapFeatNearCache(Map):
     """
     def __init__(self, client, service_name, name):
         super(MapFeatNearCache, self).__init__(client, service_name, name)
-        near_cache_config = client.config.near_cache_configs.get(name, None)
-        if near_cache_config is None:
-            raise ValueError("NearCache config cannot be None here!")
         self._invalidation_listener_id = None
-        self._near_cache = create_near_cache(client.serialization_service, near_cache_config)
-        if near_cache_config.invalidate_on_change:
+        self._near_cache = client.near_cache_manager.get_or_create_near_cache(name)
+        if self._near_cache.invalidate_on_change:
```

Comment:
Within the NearCacheManager, I get the config from the client.config.near_cache_configs using the name

Reply:
I used the near cache config within the near cache manager.  So, I removed the line which we get the near cache config and used near cache directly instead of the config. It is initialized with the given config so nothing changes